Engine/AlgorithmManager.cs

```diff
@@ -846,7 +860,11 @@ private static void HandleDelistedSymbols(IAlgorithm algorithm, Delistings newDe
                 {
                     Log.Trace("AlgorithmManager.Run(): Security delisting warning: " + delisting.Symbol.ToString());
                     var security = algorithm.Securities[delisting.Symbol];
-                    if (security.Holdings.Quantity == 0) continue;
+                    if (security.Holdings.Quantity == 0)
+                    {
+                        algorithm.RemoveSecurity(delisting.Symbol);
```

Comment:
This is wrong. Doing that, we don't allow trades on the last day the security is listed.